audio/ffmpeg.py

```diff
@@ -195,13 +196,9 @@ def _perform_ffmpeg(self, recording: corpus.Recording):
         target = os.path.join(self.out_audio_folder.get_path(), audio_filename)
         if not os.path.exists(target):
             logging.info(f"try converting {target}")
-            command_head = [
-                self.ffmpeg_binary,
-                "-hide_banner",
-                "-y",
-            ]
+            command_head = [self.ffmpeg_binary, "-hide_banner", "-y", "-threads", "1"]
             command_in = ["-i", recording.audio]
-            command_out = [target]
+            command_out = ["-threads", "1", target]
```

Comment:
is this duplicate `"-threads", "1"` intentional (e.g. ffmpeg accepts this option once for input and once for output)?

Reply:
Yes this is intentional, from my observation and some internet research this seems to be the case.